harness-delegate-ng/values.yaml

```diff
@@ -54,13 +54,13 @@ pollForTasks: "false"
 
 # Change this to alter startup probe and liveness probe settings
 startupProbe:
-  initialDelaySeconds: 10
+  initialDelaySeconds: 30
   periodSeconds: 10
-  failureThreshold: 40
+  failureThreshold: 15
 
 livenessProbe:
-  initialDelaySeconds: 30
```

Comment:
this we increased on purpose because delegate was having issues when ran on low memory. related JIRA: https://harness.atlassian.net/browse/DEL-5379 we can still decrease the values here but please run delegate on low spec and verify before merging this change.